lbz/exceptions.py

```diff
@@ -18,8 +15,8 @@ class LambdaFWException(Exception):
     message = HTTPStatus.INTERNAL_SERVER_ERROR.description
     status_code = HTTPStatus.INTERNAL_SERVER_ERROR.value
 
-    def __init__(self, message: str = ""):
-        super().__init__()
+    def __init__(self, message: str = "", errors: list = None):
```

Comment:
I wonder only where did you find that Exception accepts the message and errors 🤔  Can you share documentation or something? I see in [typeshed/builtins.pyi](https://github.com/python/typeshed/blob/7c37d6117d0141c3cabd2880e160410308f4c060/stdlib/builtins.pyi#L1378) that it just accepts `*args`, nothing else.